lib/bundle/index.js

```diff
@@ -211,16 +256,33 @@ function remap (inventory) {
   for (let entry of inventory) {
     // console.log('Re-mapping $ref pointer "%s" at %s', entry.$ref.$ref, entry.pathFromRoot);
 
-    if (!entry.external) {
+    if (!entry.inlineable && customRoots[entry.file] && customRoots[entry.file][entry.hash]) {
+      if (entry.hash === "#") {
+        set(schema, customRoots[entry.file]["#"], $Ref.dereference(entry.$ref, entry.value));
+        entry.$ref.$ref = customRoots[entry.file]["#"];
+      }
+      else {
+        let subschema = get(schema, customRoots[entry.file]["#"]);
+        let parsedHash = safePointerToPath(entry.hash);
+        let value = get(subschema, `#/${parsedHash.length === 1 ? parsedHash[0] : parsedHash.slice(0, parsedHash.length - 1).join("/")}`);
+        set(schema, customRoots[entry.file][entry.hash], parsedHash.length === 1 ? value : value[parsedHash[parsedHash.length - 1]]);
+        delete value[parsedHash[parsedHash.length - 1]];
+      }
+
+      pathFromRoot = entry.pathFromRoot;
+      hash = customRoots[entry.file][entry.hash];
+      entry.$ref.$ref = entry.pathFromRoot;
+    }
```

Comment:
Can you add a comment (or more) here? I can't figure out what it's doing. Is this where it's rewriting the pointers to point at the bundled schemas?

Reply:
Good point. I'll leave a comment explaining what's happening here.

Reply:
@wmhilton added! LMK what you think.